app/models/delivery.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: deliveries
+#
+#  id                 :integer          not null, primary key
+#  actual_arrive      :boolean
+#  description        :string
+#  details            :text
+#  supposed_to_arrive :date
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  user_id            :integer
+#
+class Delivery < ApplicationRecord
+  belongs_to :user, class_name: "User", foreign_key: "user_id"
```

Comment:
If class name and foreign key are the same, you can just write "belongs_to :user" and it'll work the same :)